new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMark.java

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes;
+
+import io.kubernetes.client.openapi.ApiException;
+import oracle.weblogic.kubernetes.annotations.IntegrationTest;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static oracle.weblogic.kubernetes.actions.TestActions.createSecret;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+@DisplayName("Simple validation of basic domain functions")
+@IntegrationTest
+class ItMark implements LoggedTest {
+
+  @Test
+  @DisplayName("Test something")
+  public void testSomeStuff() {
+    int x = 1 + 5;
+    logger.info("log message");
+
+    assertThat(true).as("Test that true is true")
+        .withFailMessage("OMG! True was not true")
+        .isEqualTo(true);
+  }
+
+  @Test
+  @DisplayName("Test something else")
+  public void testThereWasNoException() {
+    // this test will fail because it does throw an exception!
+
+    boolean result = false;
+    try {
+      result = createSecret("bob", // name
+          "weblogic", // username
+          "welcome1", // password
+          "default"); // namespace
+    } catch (Exception e) {
+      logger.info("result is " + result);
```

Comment:
Two problems: result can't be set at this line because we've caught an exception and setting result was the last step in the try block, and the stack trace of the exception is lost.